src/PermissionManager.sol

```diff
@@ -324,11 +332,12 @@ contract PermissionManager is IERC1271, Ownable, Pausable {
     ///
     /// @param permissionHash hash of the permission to revoke
     function revokePermission(bytes32 permissionHash) external {
+        // early return if permission is already revoked
         if (isPermissionRevoked[permissionHash][msg.sender]) {
-            revert RevokedPermission();
+            return;
         }
```

Comment:
Important logic change: removing revert to allow re-revoke attempts to prevent potential race conditions.